### DISCUSSION TITLE
QField Value Relation - add list entry in the field

### DISCUSSION THREAD

**Discussion by Oisin-JK** [opengisch/QField#4439] [Q&A]
I wish to use a Value Relation to select species from a list. However the list is not exhaustive, i.e. I will inevitably encounter species not in the list. I could of course use another free text attribuite to capture values not in the list, but ideally I want to be able to add to the list in the field. 

I don't think the widget can be configured to allow non-list values. Is there a way to perhaps edit the non-spatial layer directly. If I long press the list, the option 'Show Features List' takes me to the form, but there is no option to add a record.

Can anyone suggest a way to do this?

**Comment by amgiss** [comment]
You could try using a Relation Reference instead of a Value Relation. The Relation Reference widget can be configured to allow adding additional items that will appear in the dropdown list. As long as you only want a single choice from that field it should work.
